setup-scripts/init-run.sh

```diff
@@ -10,8 +10,9 @@ dnf --enablerepo=powertools install perl-Readonly -y
 # setup firewall port for 4/tcp/udp and 5001/tcp/udp
 firewall-cmd --zone=public --add-port=4/tcp --permanent
 firewall-cmd --zone=public --add-port=4/udp --permanent
-firewall-cmd --zone=public --add-port=5001/tcp --permanent
-firewall-cmd --zone=public --add-port=5001/udp --permanent
+# open port for iperf3
```

Comment:
Port numbers change. This should go into the commit message and probably into documentation. Does this need discussion?